backend/sa-mock-ai-interviewer.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Can this go in a subdirectoy inside backend package, or does it need to be outside the package
Also private key is stored in the json, probably shouldn't check this into main